docker/pypi/wmagent/bin/manage

```diff
@@ -44,13 +80,16 @@ activate_agent(){
 #
 # Database prompt so that people can poke around in the db interactively
 #
+
 db_prompt(){
```

Comment:
because of this `TODO` comment: 

https://github.com/dmwm/CMSKubernetes/pull/1451/files/a0c94024fe4a64ae487d3d0a5c0877d7c26a5da9#diff-01c5d9be01b6e6ea0ee4b12d5262b4a641bc3bdc55a79853ca90abb819996f96R43-R57 :
```
    ## TODO: To add the same functionality for recognizing the type of call, similar to _exec_oracle
    #
    # if $isPipe || $noArgs
    # then
    #     mysql -u $MYSQL_USER --password=$MYSQL_PASS -h $MYSQL_HOST --database=$wmaDBName --pager='less -SFX'
    # else
    #     local sqlStr=$1
    #     local dbName=$2
    #     if [[ -n $dbName ]]; then
    #         mysql -sN -u $MYSQL_USER --password=$MYSQL_PASS -h $MYSQL_HOST --database=$dbName --execute="$sqlStr"
    #     else
    #         mysql -sN -u $MYSQL_USER --password=$MYSQL_PASS -h $MYSQL_HOST --execute="$sqlStr"
    #     fi
    # fi
}
```